docs/content/1.index.md

```diff
@@ -149,6 +149,7 @@ export default defineNuxtConfig({
 
 Notes:
 - `'system'` is a special value; it will automatically detect the color mode based on the system preferences (see [prefers-color-mode spec](https://drafts.csswg.org/mediaqueries-5/#descdef-media-prefers-color-mode)). The value injected will be either `'light'` or `'dark'`. If `no-preference` is detected or the browser does not handle color-mode, it will set the `fallback` value.
+- Optional `colorMode.dataValue` lets you add dataset to the `html`, for example if you currently have `class="dark"` on `html`, `dataValue: 'theme'` will also set `data-theme="dark"` on `html`. This is useful when using library like daisyUI that uses `data-theme="light"` on `html` to apply theme.
```

Comment:
I don't think it is worth to be as a "note" as I see it's a regular option. We could list it in the example above with a short description.

Reply:
That would be great. I didn't know where to put it so I added it close to the other options.